test/Swashbuckle.AspNetCore.Cli.Test/ToolTests.cs

```diff
@@ -151,7 +165,22 @@ public static void Creates_New_Folder_Path()
             Assert.True(productsPath.TryGetProperty("post", out _));
         }
 
-        private static JsonDocument RunApplication(Func<string, string[]> setup, string subOutputPath = default)
+        private static string RunListCommand(Func<string, string[]> setup, string subOutputPath = default)
+        {
+            using var temporaryDirectory = new TemporaryDirectory();
+
+            var outputPath = !string.IsNullOrEmpty(subOutputPath)
+                ? Path.Combine(temporaryDirectory.Path, subOutputPath, "swagger.json")
+                : Path.Combine(temporaryDirectory.Path, "swagger.json");
+
+            string[] args = setup(outputPath);
+
+            Assert.Equal(0, Program.Main(args));
+
+            return File.ReadAllText(outputPath);
+        }
+
+        private static JsonDocument RunToFileCommand(Func<string, string[]> setup, string subOutputPath = default)
```

Comment:
This can be re-implemented in terms of calling `RunListCommand()` to avoid it being 90% the same:

```
private static JsonDocument RunToFileCommand(Func<string, string[]> setup, string subOutputPath = default)
{
    var json = RunListCommand(setup, subOutputPath);
    return JsonDocument.Parse(json);
}
```